debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -80,6 +85,10 @@ protected PostgresValueConverter(DecimalMode decimalMode, TemporalPrecisionMode
         this.schema = schema;
     }
 
+    public void setTypeInfo(TypeInfo info) {
```

Comment:
How comes this is not set via the constructor? I'd strongly prefer this class to be immutable.

Reply:
We don't have a database connection in the constructor caller AFAICT?